ksqldb-streams/src/test/java/io/confluent/ksql/execution/streams/SourceBuilderTest.java

```diff
@@ -555,6 +560,70 @@ public void shouldHandleNullKey() {
   }
 
   @Test
+  public void shouldHandleMultiField() {
```

Comment:
nit: 
```suggestion
  public void shouldHandleMultiKeyField() {
```
and similarly for the other new tests below.